app/screens/settings/display_timezone/display_timezone.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {View} from 'react-native';
+import {Edge, SafeAreaView} from 'react-native-safe-area-context';
+
+import {updateMe} from '@actions/remote/user';
+import OptionItem from '@app/components/option_item';
+import {Screens} from '@constants';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import useAndroidHardwareBackHandler from '@hooks/android_back_handler';
+import useNavButtonPressed from '@hooks/navigation_button_pressed';
+import {goToScreen, popTopScreen, setButtons} from '@screens/navigation';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {getDeviceTimezone} from '@utils/timezone';
+import {getTimezoneRegion, getUserTimezoneProps} from '@utils/user';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme) => {
+    return {
+        container: {
+            flex: 1,
+        },
+        wrapper: {
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.06),
+            flex: 1,
+            paddingTop: 35,
+        },
+        divider: {
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.1),
+            height: 1,
+        },
+        separator: {
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.1),
+            height: 1,
+            marginLeft: 15,
+        },
+        content: {
+            paddingHorizontal: 8,
+        },
+    };
+});
+
+const edges: Edge[] = ['left', 'right'];
+const SAVE_TIMEZONE_BUTTON_ID = 'save_timezone';
+
+type DisplayTimezoneProps = {
+    currentUser: UserModel;
+    componentId: string;
+}
+const DisplayTimezone = ({currentUser, componentId}: DisplayTimezoneProps) => {
+    const intl = useIntl();
+    const serverUrl = useServerUrl();
+    const timezone = useMemo(() => getUserTimezoneProps(currentUser), [currentUser.timezone]);
+    const [userTimezone, setUserTimezone] = useState(timezone);
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
+
+    const updateAutomaticTimezone = (useAutomaticTimezone: boolean) => {
+        const automaticTimezone = getDeviceTimezone();
+        setUserTimezone((prev) => ({
+            ...prev,
+            useAutomaticTimezone,
+            automaticTimezone,
+        }));
+
+        //todo: confirm with Elias
```

Comment:
what do you need to confirm?

Reply:
Please disregard this comment.  I forgot to remove it.

Reply:
so what about this?